MyWebPage/styles.css

```diff
@@ -79,3 +79,23 @@
       padding: 5px;
       margin-bottom: 5px;
     }
+
+    * {
+      margin: 0;
+    }
+    
+    footer {
+      text-align: center;
+      position: fixed;
+      bottom: 0;
+      width: 100%;
+      background-color: rgb(56, 56, 56);
+      color: white;
+      padding-top: 10px;
+      padding-bottom: 10px;
```

Comment:
You can use.
`paddding: 10px 0 10px 0;`
This style create 10px on top and bottom and 0px on left and right.